_posts/2017-12-10-trl-mesh-test.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: "TRL Mesh Test"
+slug: trl-mesh-test
+text: 'Bring your nodes, bring your computers, bring your phones, and help us set up and test a mesh at-scale'
+location: Toronto Reference Library, 789 Yonge Street  
+locationLink: http://osm.org/go/ZX6B8BWGi?way=28156063
+date: 2017-12-10
+startTime: '13:00'
+endTime: '16:00'
+---
+
+Join us at {{page.startTime | date:"%-I:%M%P "}} on {{ page.date | date: '%B %d, %Y' }} to help deploy and test the first at-scale Toronto Mesh mesh. Help us discover its capabilities, downfalls and maybe learn a thing or two you didn't know before!  Our end goal with these tests is to learn enough to deploy a Toronto Mesh at 2018's Maker Festival which happens in the Toronto Reference Library in the summer. Meet us at [{{ page.location }}]({{ page.locationLink }}) and help out. Don't forget to bring your nodes, laptops, phones and other Internet Capable devices!
```

Comment:
`s/first at-scale Toronto Mesh mesh/first at-scale Toronto Mesh network`
`s/to deploy a Toronto Mesh/to propose a Toronto Mesh deployment`
`s/Capable/capable`

Reply:
that sounds much better, thanks!